tests/python/pants_test/init/test_plugin_resolver.py

```diff
@@ -203,29 +207,37 @@ def _create_artifact(name, version, install_requires):
         )
         working_set = plugin_resolver.resolve(options_bootstrapper, complete_env, requirements)
         for dist in working_set:
-            assert (
-                Path(os.path.realpath(cache_dir)) in Path(os.path.realpath(dist.location)).parents
-            )
+            if use_uv:
+                assert ".pants.d/plugins" in os.path.realpath(dist.location)
+            else:
+                assert (
+                    Path(os.path.realpath(cache_dir))
+                    in Path(os.path.realpath(dist.location)).parents
+                )
 
         yield working_set, root_dir, repo_dir
 
 
-def test_no_plugins(rule_runner: RuleRunner) -> None:
-    with plugin_resolution(rule_runner) as (working_set, _, _):
+@pytest.mark.parametrize("use_uv", (False, True))
```

Comment:
You can parametrize a fixture instead:
```python
@pytest.fixture(params=[False, True], ids=["pip", "uv"])
def use_uv(request) -> bool:
    return request.param

def test_1(use_uv: bool): ...
def test_2(use_uv: bool): ...
```
see the docs here https://docs.pytest.org/en/6.2.x/fixture.html#parametrizing-fixtures